Tests/SwiftParserTest/AttributeTests.swift

```diff
@@ -250,15 +250,30 @@ final class AttributeTests: XCTestCase {
         """
     )
   }
-
-  func testObjcImplementationAttribute() throws {
+  
+  func testIgnoreDeprecationAttribute() throws {
     AssertParse("""
+    @available(*, deprecated)
+    class XClass {
+      var num: Int
+      init(_ num: Int) { self.num = num }
+    }
+    
+    @ignoreDeprecations(XClass)
+    func x() {
```

Comment:
It may also be worth adding some other tests for the other positions `@ignoreDeprecations` can appear in